tests/tensorflow/sota_checkpoints_eval.json

```diff
@@ -148,7 +147,7 @@
                 "mobilenet_v3_small_int8_w_sym_ch_half_a_asym_t_rb_sparsity_42": {
                     "config": "examples/tensorflow/classification/configs/sparsity_quantization/mobilenet_v3_small_imagenet_rb_sparsity_int8.json",
                     "reference": "mobilenet_v3_small",
-                    "target": 67.55,
+                    "target": 67.59,
```

Comment:
@dongkwan-kim01 Could you please also update the metric value in [this](https://github.com/openvinotoolkit/nncf/blob/develop/README.md) file?

Reply:
Yes, updated it!